src/styles/card.scss

```diff
@@ -0,0 +1,111 @@
+.catalog {
+  max-width: getWidth($width, $gap, $index);
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin: auto;
+
+  &__card {
+    margin-right: 50px;
+
+    &:nth-child(-n+4) {
+      padding-top: 51px;
+      margin-bottom: 48px;
+
+    }
+    &:nth-child(4n) {
+      margin-right: 0;
+     
+    }
+  }
+
+  &:nth-last-child(-n + 4) {
+    padding-bottom: 51px;
+  }
+  
+}
+
+.card {
+  
```

Comment:
```suggestion
```

Reply:
fix everywhere, there is no need in the space at the start and at the end of the block